pkg/controller/storagecluster/reconcile.go

```diff
@@ -913,6 +913,15 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, r
 	} else {
 		reqLogger.Info(fmt.Sprintf("No monDataDirHostPath, monPVCTemplate or storageDeviceSets configured for storageCluster %s", sc.GetName()))
 	}
+	multusNetwork := sc.Spec.Network
+	multusNetworkNil := ocsv1.NetworkSpec{}
+	multusSelector := make(map[string]string)
+	if multusNetwork != multusNetworkNil {
```

Comment:
When we provide some random value in `sc.Spec.Network`, this condition is met and we use multus. Shouldn't we validate the user input?

Reply:
Do you mean fetching the NetworkAttachmentDefinition object by the value specified by the user and confirming it?
Well, it makes sense to validate this way.

Reply:
The NAD is presented by the UI so this is safe, we will get something that exists. Later on Rook-Ceph will validate the NAD too. Not sure if we need to duplicate that validation "just" for the manual bootstrap case. However, I can see that making sense.

Reply:
Well, if we validate we make it more reliable and an error can be thrown even before the operator goes and creates CephCluster. 
Validation can be part of a followup PR.